Tools/machines/karolina-it4i/karolina_gpu_warpx.profile.example

```diff
@@ -48,8 +48,8 @@ fi
 
 # PBS is not well configured to properly address GPU visibility
 # per MPI rank. Thus, we define the node layouts here.
-export GPUS_PER_SOCKET=2
-export GPUS_PER_NODE=4
+export GPUS_PER_SOCKET=4
```

Comment:
@ax3l , I am wondering, where are we using these 2 variables? Also, do you agree with the pinning solution offered by IT4I in [this gist](https://gist.github.com/berceanu/f9e8118c508ad9a23df1f7bb80c1c9a9#gpu_bindsh)?